plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -655,29 +659,29 @@ public void close()
 
     private static Iterator<?>[] getIterators(Iterable<?>[] values)
     {
-        return stream(values)
+        return Stream.of(values)
```

Comment:
nit: migration Arrays.stream -> Stream.of could be a separate preparatory commit.